lib/Dialect/FIRRTL/Transforms/InferResets.cpp

```diff
@@ -40,10 +40,14 @@ using namespace firrtl;
 // Utilities
 //===----------------------------------------------------------------------===//
 
+namespace {
```

Comment:
I don't think there is any value in putting these in a namespace.  The global types are constrained to the compilation unit.

Reply:
This was a bad implementation choice to get around the name collision with `InstancePath` that is added by the additions to `InstanceGraph`. I'll changed the colliding thing from `InstancePath = SmallVector<...>` to `InstancePathVec`.